### PR TITLE
Fix types issues after PostCSS update

### PR DESCRIPTION
**Pull request by hudochenkov**
> Which issue, if any, is this issue related to?

Related to #4942.

### CODE REVIEW

File: lib/postcssPlugin.js [hunk 1]
```diff
@@ -1,9 +1,9 @@
+// @ts-nocheck
```
Comment (hudochenkov) [inline lib/postcssPlugin.js:1]: Types from PostCSS don't have `postcss.plugin` type anymore.

This file should be rebuild with visitors API anyway, so ignoring the problem for now.

File: lib/__tests__/disableRanges.test.js [hunk 2]
```diff
@@ -955,7 +955,8 @@ it('Less // line-disabling comment (with description)', async () => {
 	});
 });
 
-it('Less // disable next-line comment (with multi-line description)', async () => {
+// eslint-disable-next-line jest/no-disabled-tests -- TODO: investigate the reason
+it.skip('Less // disable next-line comment (with multi-line description)', async () => {
```
[the remark this answers is not in the document]
Comment (jeddy3) [reply lib/__tests__/disableRanges.test.js:959]: I'll create a low priority follow-up issue for this as I doubt this affects many users if any.

**Review by jeddy3 — APPROVED** [review]
Thanks, LGTM!

I say go ahead and merge into the `update-to-postcss-8` branch to keep the ball rolling.

File: lib/postcssPlugin.js (hunk 1 above)
Comment (jeddy3) [reply lib/postcssPlugin.js:1]: Sounds good.

I believe this is how CSSNano pulls together its PostCSS@8 plugin pack:

https://github.com/cssnano/cssnano/blob/b60f54bedafe3781ff58f0888ab45ff5c56aee09/packages/cssnano/src/index.js#L91-L132

**Review by ybiquitous — APPROVED** [review]
LGTM! 👍🏼

**Pull request merged**